reference/library/src/morpheus-master/sass/base/_icons.scss

```diff
@@ -6,10 +6,12 @@
 	text-align: center;
 }
 
-
 @if $skin-with-icons {
-	[class^="icon-sakai"],
-	[class*=" icon-sakai"] {
```

Comment:
Hi @ern.  Is there a particular reason why we are removing the generic selectors?  These were added so we could easily include Sakai icons in any context without having to add to this selector.  I believe some tools that assume these generic selectors will now fail to show icons, including GradebookNG and Site Setup's Tool Order.